app/main.py

```diff
@@ -1 +1,20 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import VaccineError, NotWearingMaskError
+
+
+def go_to_cafe(friends: list, cafe: str):
+    selected_cafe = Cafe(cafe)
+    some_arr = []
+    masks_to_buy = 0
+    for person in friends:
+        try:
+            selected_cafe.visit_cafe(person)
+            some_arr.append(person)
```

Comment:
there is no need to add person to the list and do some checking. Because if no Vaccine errors happen it already means that friends can go to cafe